test/e2e/scheduling/nvidia-gpus.go

```diff
@@ -118,7 +118,7 @@ func getGPUsAvailable(f *framework.Framework) int64 {
 			gpusAvailable += (&val).Value()
 		}
 	}
-	return gpusAvailable
+	return gpusAvailable / 2
```

Comment:
Instead of hardcoding 2 here, can we still return available gpus from this function, and then in testNvidiaGPUs, we get pod from makeCudaAdditionDevicePluginTestPod(), and count how many nvidia gpus would be required by that pod?

Reply:
+1

Reply:
Thanks for the suggestion. That seems better. Updated